### ISSUE TITLE
Fail to run batch runner scenario from save scenario tool

### ISSUE THREAD

**Issue by myarjunar** [inasafe/inasafe#3955]
### Problem
cannot find path generated from save scenario

### Expected Behaviour
path generated from save scenario tool is recognized by batch runner

### Environment
* InaSAFE : 4.0 develop
* QGIS : 2.14
* OS : Windows
* Data used (filenames and paths so as to reproduce, a project is better):
```
[Flood new scenario] 
exposure = E:\Sync\data.inasafe.org\InaSAFE 3.5 Training Data\DKI Jakarta\DKI Jakarta\02 Exposure Data\Jakarta_roads_WGS84.shp 
hazard = E:\gisdata\Jakarta\JakartaFloods-2April2016\flood_data.json 
extent = 106.803668, -6.325429, 106.886219, -6.237527 
extent_crs = EPSG:4326 
aggregation = E:\Sync\data.inasafe.org\InaSAFE 3.5 Training Data\DKI Jakarta\DKI Jakarta\03 Aggregation Data\Jakarta_Subdistrict_Boundary_WGS84.shp
```

**Comment by myarjunar** [comment]
@Gustry @timlinux do we support json format for layer?

**Comment by timlinux** [comment]
Yes sure - any valid QGIS supported format should be able to work....

**Comment by myarjunar** [comment]
closed in #3966
